### DISCUSSION TITLE
Passaggio all'anno successivo

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#280] [Richieste di aiuto] [answered]
Salve, chiedo come è possibile effettuare il passaggio all'anno successivo.

**Comment by manuelbarberi** [comment]
Devo archiviare tutto e dopo... reinstallare tutto? Come se fosse un registro nuovo... uso però lo stesso database.

**Comment by trinko** [comment, marked as answer]
Si, la procedura attuale prevede di fare una nuova installazione vedi:
[Passaggio all'anno successivo](https://github.com/trinko/giuaschool/discussions/269#discussion-4135406)

Ma conto di fare, per l'inizio del nuovo anno, una procedura che permetta di fare tutto (quasi) in automatico, riutilizzando lo stesso database ripulito.

**Reply by manuelbarberi** [reply]
Usando lo stesso database posso pulire tutto e reinstallare il registro importando i dati vecchi.

**Reply by trinko** [reply]
> Usando lo stesso database posso pulire tutto e reinstallare il registro importando i dati vecchi.

Si, è una buona soluzione.
